### PR TITLE
Initial Implementation 

### PR DESCRIPTION
**Pull request by kooba**
Example of cross-language integration between Nodejs gateway and Nameko services via GRPC endpoints.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+build-examples-base:
+	docker build -t nameko-examples-grpc-base -f docker/base.docker docker;
+
+build-wheel-builder: build-examples-base
+	docker build -t nameko-examples-grpc-builder -f docker/build.docker docker;
+
+run-wheel-builder: build-wheel-builder
+	for image in $(PYTHON_IMAGES) ; do \
+	make -C $$image run-wheel-builder; \
+	done
```
Comment (mattbennett) [inline Makefile]: Might be nicer to use a single Dockerfile with build stages now, e.g. https://github.com/Overseas-Student-Living/dev-stats/blob/master/deploy/Dockerfile

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	--proto_path=proto \
+	--python_out=orders/orders \
+	--grpc_python_out=orders/orders \
+	orders.proto
+	@# Hack untill I figure how to invoke this piece of code:
+	@# https://github.com/grpc/grpc/pull/10862/files
+	@sed -i.bak 's/^\(import.*_pb2\)/from . \1/' orders/orders/*grpc.py
+	@rm orders/orders/*.bak
+
+.PHONY: proto
```
Comment (mattbennett) [inline Makefile]: Think this normally goes at the top of the Makefile, so it's easy to find if you want to add another rule.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+proto: products-proto orders-proto
+
+# Relies on `nodemon` nodejs utility installed globally:
+# `$ sudo npm install -g nodemon --unsafe-perm=true --allow-root`
+
+develop-orders: proto
+	nodemon --ext py --watch orders/orders --watch orders/config.yaml \
+	--exec "nameko run --config orders/config.yaml orders.service"
```
Comment (mattbennett) [inline Makefile:70]: This is very nice. How does it behave when it reloads? Graceful stop and then restart, or just kill?

**Review by mattbennett — APPROVED** [review]
This is amazing! A distillation of so many good ideas :)

File: Makefile (hunk 3 above)
Comment (kooba) [reply Makefile:70]: It works absolutely great. No hick-ups on restart. I believe it does send kill signal by default but that can be controlled: https://github.com/remy/nodemon#controlling-shutdown-of-your-script
Don't think that would be a problem in development though.

**Pull request merged**